_layouts/docs.html

```diff
@@ -36,7 +36,7 @@
 <meta name="twitter:card" content="summary_large_image">
 <meta name="twitter:site" content="@docker_docs">
 <meta name="twitter:url" content="https://twitter.com/docker_docs">
-<meta name="twitter:title" content="Docs team">
+<meta name="twitter:title" content="Docker docs">
```

Comment:
Yeah I don't think the outcome will be much better unless we somehow include the page title. Maybe @johndmulhausen knows how to do it.